### PR TITLE
Feature/Label-Source-Addition

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

This PR adds the `label` source table. This was necessary as the `issue_label` depreciated the `label` column. The respective label information will now be synced within the `label` table. 

Additionally, a few minor updates were made to the `issue_label` table and some spelling updates were applied.

**Does this PR introduce a breaking change?**

- [X] Yes (please provide breaking change details below.)
- [ ] No  (please provide explanation how the change is non breaking below.)

Since this PR removes a field from the `issue_label` table and adds a new source table, this will be reflected as a breaking change.

**Is this PR in response to a previously created Issue**

- [X] Yes, Issue #21 
- [ ] No 

**How did you test the PR changes?** 

 
- [X] CircleCi  
- [ ] Other (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 
- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🏷️ 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# dbt_github_source v0.5.0
+
+## Fixes
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:3]:
```suggestion
## 🚨  Breaking Changes 🚨 
```

**Review by fivetran-jamie — APPROVED** [review]
lgtm

**Comment by fivetran-jamie** [comment]
ah also remember to change the package version in the README installation instructions since it's breaking

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# dbt_github_source v0.5.0
+
+## 🚨  Breaking Changes 🚨 
+- Addition of the `label` source table. This was necessary as the `issue_label` depreciated the `label` column. The respective label information will now be synced within the `label` table. ([#22](https://github.com/fivetran/dbt_github_source/pull/22))
```
Comment (fivetran-jamie) [inline CHANGELOG.md:4]: also might wanna add the same note about kicking off historical resyncs in the source changelog?

Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:4]: Great idea!

**Pull request merged**